hs_app_netCDF/templates/pages/netcdfresource.html

```diff
@@ -4,110 +4,102 @@
-                                    <tr><th class="text-muted">Datum </th><td>{{ original_coverage.datum}}</td></tr>
-                                {% endif %}
-
-                                {% if original_coverage.projection_string_type %}
-                                    <tr><th class="text-muted">Coordinate String Type </th><td>{{ original_coverage.projection_string_type}}</td></tr>
-                                {% endif %}
-
-                                {% if original_coverage.projection_string_text %}
-                                    <tr><th class="text-muted">Coordinate String </th><td style="word-break:break-all" >{{ original_coverage.projection_string_text}}</td></tr>
-                                {% endif %}
-
-
-                            </tbody>
-                        </table>
+    <div class="col-md-12">
+        {# Add the view mode html code below for resource extended meta tab #}
+
+        {% if extended_metadata_exists %}
+            {# Add the original coverage meta #}
+            <legend>Spatial Reference</legend>
+            {% if original_coverage %}
+                {% if original_coverage.projection or original_coverage.projection_string_type or original_coverage.projection_string_text %}
+                <table class="custom-table">
+                    <tbody>
+                        {% if original_coverage.projection %}
+                            <tr><th class="text-muted">Coordinate Reference System </th><td>{{ original_coverage.projection}}</td></tr>
                         {% endif %}
 
-                        <h4>Extent</h4>
+                        {% if original_coverage.datum %}
+                            <tr><th class="text-muted">Datum </th><td>{{ original_coverage.datum}}</td></tr>
+                        {% endif %}
 
-                        <table class="custom-table">
+                        {% if original_coverage.projection_string_type %}
+                            <tr><th class="text-muted">Coordinate String Type </th><td>{{ original_coverage.projection_string_type}}</td></tr>
+                        {% endif %}
 
-                            {% if original_coverage.northlimit %}
-                                <tr><th class="text-muted">North </th><td>{{ original_coverage.northlimit }}</td></tr>
-                            {% endif %}
+                        {% if original_coverage.projection_string_text %}
+                            <tr><th class="text-muted">Coordinate String </th><td style="word-break:break-all" >{{ original_coverage.projection_string_text}}</td></tr>
+                        {% endif %}
+                    </tbody>
+                </table>
+                {% endif %}
 
-                            {% if original_coverage.eastlimit %}
-                                <tr><th class="text-muted">East </th><td>{{ original_coverage.eastlimit }}</td></tr>
-                            {% endif %}
+                <h4>Extent</h4>
```

Comment:
Why not use <legend> here?

Reply:
We could, I'll try to make this change after the release. For now we'll leave it like this since the only difference is a bit of spacing.